lib/animas/animasSimulator.js

```diff
@@ -74,6 +74,21 @@ exports.make = function(config){
     return e;
   }
 
+  function validateDuration(basal) {
```

Comment:
I'd prefer a function name like `truncateDuration` since this can mutate the basal object passed in.